build.gradle

```diff
@@ -30,19 +30,13 @@ import org.opensearch.gradle.test.RestIntegTestTask
 buildscript {
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = '7.10.3'
+        opensearch_version = '1.0.0'
     }
 
     repositories {
+        mavenLocal()
         mavenCentral()
         maven { url "https://plugins.gradle.org/m2/" }
-        maven {
```

Comment:
By removing this private S3 maven, can we use `./gradlew build` to build AD plugin locally?

Reply:
Yup. Free at last!

Reply:
Btw the S3 credentials being required could also be worked around by tagging `?: ''` to them.

Reply:
After removing this s3 maven, I think we need build all of other dependencies and publish them to local maven repo in our laptop, just similar as the CI workflow to make the local ```./gradlew build``` work, right?

Reply:
Correct, run `./gradlew publishToMavenLocal` on each dependency once.